segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -179,7 +195,7 @@ namespace pcl
         */
       ExcludeLabelSetConstPtr exclude_labels_;
 
-      float distance_threshold_;
+      float distance_threshold_ = 0.005f;
 
       bool depth_dependent_;
```

Comment:
true or false? I know it didn't have a default before, but can we add one (false would be like 0-init)? It'll be less headaches in future

Reply:
I just anticipated it would be false, but I'll add it explicitly 👍